nekoyume/Assets/_Scripts/Game/ScriptableObject/BonusBuffIconData.cs

```diff
@@ -8,12 +9,17 @@ namespace Nekoyume.Game.ScriptableObject
     public class BonusBuffIconData
     {
         [SerializeField]
-        private SkillCategory skillCategory;
+        [EnumToString(typeof(SkillCategory))]
+        private string skillCategory;
 
         [SerializeField]
         private Sprite iconSprite;
 
-        public SkillCategory SkillCategory => skillCategory;
+        public SkillCategory SkillCategory()
+        {
+            Enum.TryParse(typeof(SkillCategory), skillCategory, true, out var skillCategoryEnum);
+            return (SkillCategory)skillCategoryEnum;
+        }
```

Comment:
```suggestion
        public SkillCategory SkillCategory
        {
            get
            {
                Enum.TryParse(typeof(SkillCategory), skillCategory, true, out var skillCategoryEnum);
                return (SkillCategory) skillCategoryEnum;
            }
        }
```
이렇게 사용하는편이 추가수정사항을 덜만들것같은데 확인부탁드립니다~!

Reply:
좋은거같습니다~

Reply:
[reflect on review for reduce diff](https://github.com/planetarium/NineChronicles/pull/4574/commits/fa502011c853ba0a9512b52839488d15c34a8bad)
여기서 TryParse 결과에 따라 적당히 반환해주게 반영해봤습니다 @jonny-jeahyunchoi